app/partials/app.cjsx

```diff
@@ -19,9 +19,12 @@ PanoptesApp = React.createClass
     geordi: @geordiLogger
 
   getEnv: ->
+    shell_env = process.env.NODE_ENV
     reg = /\W?env=(\w+)/
     browser_env = window?.location?.search?.match(reg)
-    @state?.env || browser_env?[1] || 'staging'
+    env = shell_env || browser_env?[1] || 'staging'
```

Comment:
minor but I think this could just be one line, `env = process.env.NODE_ENV || browser_env?[1] || 'staging'`

I think the intention is clear enough without the need for the extra variable but i'm about a 3/10 on this.